.github/workflows/run_check.yml

```diff
@@ -23,6 +23,26 @@ permissions:
 # Note: consider the job names to be global if you want to require
 # them to succeed for a PR. :)
 jobs:
+  lint-and-format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+      - name: Install Poetry
```

Comment:
This should be "uv" instead of poetry. We have migrated, but the poetry.lock is being kept for a while.

Reply:
ok